kdumpctl

```diff
@@ -1051,6 +1051,83 @@ check_final_action_config()
 	esac
 }
 
+get_recommend_crashkernel_value()
+{
```

Comment:
I think `get_recommend_crashkernel_value`  is duplicating `kdump_get_arch_recommend_size`. Btw, there is also a problem with `get_recommend_crashkernel_value` i.e it uses MemTotal from /proc/meminfo which is actually smaller than the system RAM.

Reply:
I will remove it and resort to kdump_get_arch_recommend_size(). Thanks